src/components/dropdown.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Person } from '../types/Person';
+
+interface Props {
+  people: Person[];
+  onSelect?: (person: Person) => void;
+}
+
+export const Dropdown: React.FC<Props> = (
+  {
+    people,
+    onSelect = () => { },
+  },
+) => (
+  <div className="dropdown-menu" role="menu">
+    <div className="dropdown-content">
+      {!people.length ? (
+        <div className="dropdown-item">
+          <p>
+            No matching suggestions
+          </p>
+        </div>
+      ) : people.map(person => (
+        // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions
```

Comment:
Try to fix eslint rule and remove comment

Reply:
Там не можливо пофіксити, бо я задаю onClick  діву, а це не інтерактивний елемент

Reply:
I believe you) the idea was for you to rewrite it to a select with options or an input or some other interactive element. For this task, this is only a recommendation, but there are situations when it is necessary and it would be good if you knew how to do it